src/test/scala/mesosphere/marathon/api/validation/V2AppDefinitionValidatorTest.scala

```diff
@@ -1,234 +0,0 @@
-package mesosphere.marathon.api.validation
```

Comment:
@aquamatthias Sorry for late reply. As I was deleting the whole `validation` folder with annotation validation, I did not pay attention, that there was also one test included in there. Sorry for that, I added it back again adjusting it to new validation model.